File: Codeforces/Codeforces Round #787/A_Food_for_Animals.cpp
#include "bits/stdc++.h"
using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int t;
    cin >> t;
    while(t--){
        int a, b, c, x, y;
        cin >> a >> b >> c >> x >> y;
        int needed_dog_food = max(0, x-a);
        int needed_cat_food = max(0, y-b);
        string ans = c >= needed_cat_food + needed_dog_food ? "YES" : "NO";
        cout << ans <<"\n";
    }

}

File: Codeforces/Codeforces Round #787/A_Food_for_Animals.py
import sys
input = sys.stdin.readline

t = int(input())
for _ in range(t):
    a, b, c, x, y = map(int, input().split())
    needed_dog_food = max(0, x-a)
    needed_cat_food = max(0, y-b)
    answer = "YES" if c >= needed_cat_food + needed_dog_food else "NO"
    print(answer)
